servers/republik/__tests__/helpers.js

```diff
@@ -33,26 +34,28 @@ const createLocalApolloFetch = () => {
     })
 }
 
+let pgdb
 const connectIfNeeded = async function () {
   if (server) return server
   await redis.flushdbAsync()
   await sleep(1000)
-  server = await Server.run()
+  server = await Server.start()
+  pgdb = await PgDb.connect()
   return server
 }
 
-const pgDatabase = () => server.pgdb
+const pgDatabase = () => pgdb
 
 const disconnect = async () => {
   await Server.close()
 }
 
 const throwOnOpenTransaction = async () => {
-  const locksEnd = await server.pgdb.query('SELECT count(*) FROM pg_stat_activity WHERE state = :state', {
+  const locksEnd = await pgDatabase().query('SELECT count(*) FROM pg_stat_activity WHERE state = :state', {
     'state': 'idle in transaction'
   })
   if (locksEnd && locksEnd[0] && locksEnd[0].count > 0) {
-    const locks = await server.pgdb.query('SELECT * FROM pg_stat_activity WHERE state = :state', {
+    const locks = await pgDatabase.query('SELECT * FROM pg_stat_activity WHERE state = :state', {
```

Comment:
```
pgDatabase.query(...)
```

should be

```
pgDatabase().query(...)
----------^
```

Assuming `pgDatabase` [is a function](https://github.com/orbiting/backends/pull/36/files#diff-cb0d8592dfc82623ed878f79c62ca6dfR47) this line will break.